### ISSUE TITLE
Support parsing SQL Server multi-column ADD syntax

### ISSUE THREAD

**Issue by LutzHeinrichs** [jOOQ/jOOQ#10098]
### SQL translator incorrectly translates this query:

```sql
ALTER TABLE [dbo].[ImportSetups]
ADD 
FolderImportErrorFlag bit DEFAULT(0) NOT NULL,
FolderImportErrorCount smallint DEFAULT(50) WITH VALUES,
FolderImportErrorThreshold smallint DEFAULT(20) WITH VALUES,
FolderImportMinSampleSize smallint DEFAULT(100) WITH VALUES,
FolderImportPath nvarchar(255) DEFAULT('Folder_Import\') WITH VALUES,
FolderRejectPath nvarchar(255) DEFAULT('Folder_Reject\') WITH VALUES,
FolderResultPath nvarchar(255) DEFAULT('Folder_Result\') WITH VALUES

**Comment by lukaseder** [comment]
Thanks a lot for your report. We currently only support these two syntax variants for multi column additions:

```sql
-- Oracle style
ALTER TABLE t ADD (col1 INT, col2 INT);

-- PostgreSQL style
ALTER TABLE t ADD col1 INT, ADD col2 INT;
```

Also, we currently don't correctly parse (nor translate) the `WITH VALUES` clause: #10115. We could probably parse and ignore it for now. For example, in MySQL, Oracle, PostgreSQL, `WITH VALUES` is always implicit.

**Comment by lukaseder** [comment]
Fixed for jOOQ 3.14.0. Backport scheduled for 3.13.2 (#10117).

Thanks again for reporting. The fix will be available from the https://www.jooq.org/translate soon.

**Comment by LutzHeinrichs** [comment]
Sweet, thanks for getting back to me, and actually putting a fix in 😊

Best,
Lutz

Lutz Heinrichs
Software Engineer IV
Vertex Inc. | 2301 Renaissance Blvd. | King of Prussia, PA, 19406
Phone: 484.595.6341 Mobile: 484.320.0818
lutz.heinrichs@vertexinc.com
[cid:image001.png@01D61A1B.D53619A0]
[Follow us on LinkedIn]<http://www.linkedin.com/company/276981?trk=tyah>  [Follow us on Facebook] <https://www.facebook.com/pages/Vertex-Inc/198291610201908>   [Follow us on Twitter] <https://twitter.com/vertexinc>
